packages/st2/Makefile

```diff
@@ -53,7 +53,7 @@ ifeq ($(DEBIAN),1)
 endif
 
 wheelhouse: .stamp-wheelhouse
-.stamp-wheelhouse: populate_version requirements
+.stamp-wheelhouse: | populate_version requirements
```

Comment:
this a order only dependency, here's some info https://www.gnu.org/software/make/manual/html_node/Prerequisite-Types.html